code/modules/mob/living/living_defines.dm

```diff
@@ -68,3 +68,5 @@
 
 	var/mob/living/mental_dominator //The person controlling the mind of this person, if applicable
 	var/mob/living/mind_control_holder/mind_control_holder //If the mob is being mind controlled, where their old mind is stored (check clock_mobs.dm)
+
+	var/obj/effect/proc_holder/ranged_ability //Any ranged ability the mob has, as a click override

```

Comment:
nnnnghhh adding a var to all living mob just for a tiny feature that only xeno use....nnghhghg